.github/workflows/pull_request_check.yml

```diff
@@ -0,0 +1,71 @@
+name: Perform unit test on the new pull requests.
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  project-setup-and-run-tests:
+    runs-on: ubuntu-latest
+
+     # Service containers to run with `runner-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
```

Comment:
Does this image support `postgis`? We might have to use another image that comes with `postgis` installed.

Reply:
I'll search about this and do the needful.